script/create-dist.py

```diff
@@ -40,7 +45,19 @@ def main():
 
     with scoped_cwd(base_dir):
         if output.endswith('.zip'):
-            make_zip(output, inputs, dir_inputs)
+            try:
+                make_zip(output, inputs, dir_inputs)
+            except RuntimeError as e:
+                if sys.platform != 'darwin':
+                    raise
+                # create_dist_zips is currently flaky on macOS. Warn about the
```

Comment:
please make a follow up task to fix this.

Reply:
I would use https://github.com/brave/brave-browser/issues/41085